cartridges/int_adyen_overlay/cartridge/js/pages/checkout/adyen-cse.js

```diff
@@ -16,49 +16,33 @@ function pad ( number ) {
  * @description Initializes Adyen CSE  Billing events
  */
 function initializeBillingEvents() {
-
+	
 	$('#billing-submit').on('click', function (e) {
 		var radioVal = $('.payment-method-options').find(':checked').val();
 		if ('CREDIT_CARD' == radioVal){
+			
+			if(!window.CardValid) {
+				return false;
+			}
+			
+			var card = window.AdyenCard;
+		
+			$('#dwfrm_billing_paymentMethods_creditCard_type').val(card.state.brand);
```

Comment:
Minor L31 to L36: We can make it more clean by moving it to another function to copyData.